src/views/v2/commerce/ticket/regular-price.php

```diff
@@ -23,4 +23,11 @@
 	return;
 }
 
-echo esc_html( $price->get_currency() );
+$regular_price_label = $price->get_currency();
+
+// If the price is zero, we should display it as free.
+if ( $price->get_decimal() == 0 ) {
+	$regular_price_label = esc_html__( 'Free', 'event-tickets' );
```

Comment:
```suggestion
	$regular_price_label = esc_html_x( 'Free', "Text displayed for a free ticket.", 'event-tickets' );
```
Or etc...add context.